homeassistant/components/deconz/__init__.py

```diff
@@ -39,6 +39,8 @@
 
     await gateway.async_update_device_registry()
 
+    await async_add_uuid_to_config_entry(hass, config_entry)
```

Comment:
You might want to put the guard outside of the `await`, so you don't switch tasks.